Model/Config.php

```diff
@@ -96,7 +96,7 @@ public function isActive(): bool
      */
     public function isMerchantValid(): bool
     {
-        return $this->scopeConfig->isSetFlag(self::XML_CONFIG_PATH_MERCHANT_ID_VALID, ScopeInterface::SCOPE_WEBSITE);
+        return true; //$this->scopeConfig->isSetFlag(self::XML_CONFIG_PATH_MERCHANT_ID_VALID, ScopeInterface::SCOPE_WEBSITE);
```

Comment:
I think this might have been a debugging thing and should actually be coming from the config right?